specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+            },
+            "authCode": {
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DomainAvailabilityCheckResult": {
+      "description": "Domain availability check result.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the domain.",
+          "type": "string"
+        },
+        "available": {
+          "description": "<code>true</code> if domain can be purchased using CreateDomain API; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "domainType": {
+          "description": "Valid values are Regular domain: Azure will charge the full price of domain registration, SoftDeleted: Purchasing this domain will simply restore it and this operation will not cost anything.",
+          "enum": [
+            "Regular",
+            "SoftDeleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DomainType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "DomainCollection": {
+      "description": "Collection of domains.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Domain"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainControlCenterSsoRequest": {
+      "description": "Single sign-on request information for domain management.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "URL where the single sign-on request is to be made.",
+          "type": "string",
+          "readOnly": true
+        },
+        "postParameterKey": {
+          "description": "Post parameter key.",
+          "type": "string",
+          "readOnly": true
+        },
+        "postParameterValue": {
+          "description": "Post parameter value. Client should use 'application/x-www-form-urlencoded' encoding for this value.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainOwnershipIdentifier": {
+      "description": "Domain ownership Identifier.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "DomainOwnershipIdentifier resource specific properties",
+          "type": "object",
+          "properties": {
+            "ownershipId": {
```

Comment:
>ownershipId [](http://example.com/codeflow?start=13&length=11)

Add details to the description about the format (e.g. Guid, integer, name, Azure resourceId). If it's a resource ID include resourceId in the name, e.g. ownerResourceId.